packages/viewer/src/interactions/keyStateInteractionHandler.ts

```diff
@@ -0,0 +1,47 @@
+import { EventDispatcher, Listener, Disposable } from '@vertexvis/utils';
+import { InteractionHandler } from './interactionHandler';
+
+export class KeyStateInteractionHandler implements InteractionHandler {
+  private pressed: Record<string, boolean> = {};
+
+  private keyStateChange = new EventDispatcher<Record<string, boolean>>();
+
+  public constructor() {
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.handleKeyUp = this.handleKeyUp.bind(this);
+  }
+
+  public initialize(): void {
+    window.addEventListener('keydown', this.handleKeyDown);
```

Comment:
Any benefit here to using pointer events? I don't think it'd matter much if this is a specific key-binding event.

Reply:
I don't think the keyboard would dispatch pointer events, and we'll have to use `keydown`/`keyup`, if that's what you were thinking of replacing here?

Reply:
Haha, yeah. Disregard. 🤦  It's not a mouse down :D